src/zkevm_specs/evm/execution/__init__.py

```diff
@@ -5,5 +5,6 @@
 from .push import *
 from .jump import *
 from .jumpi import *
+from .block_coinbase import *
```

Comment:
Could we name it after `ExecutionState` (e.g. `coinbase.py`) ?

Reply:
for `coinbase` case is ok, but I consider there are others block's op code (`time`, `number` etc), if use file name directly like `time.py`, `number.py` , easily cause ambiguous meaning comparing to system files.
So I prefer to keep prefix for all , like `block_coinbase.py`, `block_time.py`, `block_number.py` etc.

Reply:
I see, it makes sense to me.